modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ApexClientCodegen.java

```diff
@@ -46,7 +47,7 @@
     private String classPrefix = "OAS";
     private String apiVersion = "42.0";
     private String buildMethod = "sfdx";
-    private String namedCredential = classPrefix;
+    private String namedCredential;
```

Comment:
@loganm What about keeping the default as `classPrefix`?

Reply:
The current behaviour of the generator is to use the spec title for the namedCredential. For example, I'm using a Dun and Bradstreet spec with the title 'Search_for_a_Company_or_Contact'.

The default classPrefix is 'OAS' which doesn't mean much. The spec title is more informative about purpose of the named credential.

Now that the namedCredential var is being referenced, leaving the default value as classPrefix here would also change the default behaviour of the generator. It could be a breaking change for anyone using it presently.


Do you think it's possible to get at the spec title as the default here instead of the class prefix?

Reply:
> Now that the namedCredential var is being referenced, leaving the default value as classPrefix here would also change the default behavior of the generator. It could be a breaking change for anyone using it presently.

Yes, we're on the same page to make this change backward-compatible.

Let me review again as I thought the default value is no longer the same.

Reply:
I just updated the petstore samples and I can see the issue you've fixed in this PR 👍